### DISCUSSION TITLE
Bar Chart Daily Aggregations

### DISCUSSION THREAD

**Discussion by Frankenberrypi** [dbuezas/lovelace-plotly-graph-card#68] [Q&A]
I have sensor entity that stores the daily amount of rain and resets every midnight.  I'd like to have a bar graph with one bar for each day that has the maximum (or last) value for each day.  This is probably possible with a fairly short lambda function, but I have no idea how to write it.  This is what I have so far:
`type: custom:plotly-graph
hours_to_show: 168
layout:
  height: 400
  xaxis:
    rangeselector:
      'y': 1.25
      buttons:
        - count: 7
          step: day
        - count: 1
          step: month
entities:
  - entity: sensor.rain_day
    name: Daily Rain
    type: bar
refresh_interval: 10
`
![image](https://user-images.githubusercontent.com/8370483/187097679-7bc0640f-ff01-40ed-bd89-eca90179aaa1.png)

...and I'm hoping for something that looks about like this:
![image](https://user-images.githubusercontent.com/8370483/187097721-878fbf50-450f-4362-a171-5336345ace60.png)

So far I've tried a few lambda functions with things like math.max, but I don't know how to restrict that to per day.  Plotly appears to have an [aggregation tool](https://plotly.com/javascript/aggregations/) which should work, but I'm not sure if it's implemented or how to put it in the yaml.

**Comment by dbuezas** [comment]
I think #28  will point you in the right direction. If you need assistance let me know and I'll give you a hand :)
